tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter.cc

```diff
@@ -260,9 +260,7 @@ StatusOr<std::unique_ptr<HloInstruction>> TryRewriteToCudnnForwardRelu(
 
 // Fuse bias/scaling/ReLU with convolution custom call with floating point
 // output
-StatusOr<bool> RunFuseBiasSideActivation(
-    HloModule* module,
-    std::unordered_set<const HloInstruction*>& tracked_custom_calls) {
```

Comment:
We don't need tracked_custom_calls.  In the previous commit, I used it to track partially matched convolution, especially those with float output.